central/debug/service/diagnostics.go

```diff
@@ -141,6 +144,20 @@ func pullCentralClusterDiagnostics(ctx context.Context, filesC chan<- k8sintrosp
 	}
 }
 
+func getK8sInClusterConfig() (*rest.Config, error) {
+	restCfg, err := rest.InClusterConfig()
+	if err != nil {
+		return nil, err
+	}
+	// Replacing raw IP address with kubernetes.default.svc
+	// allows for easier proxy configuration.
+	if env.ManagedCentral.BooleanSetting() {
+		port := os.Getenv("KUBERNETES_SERVICE_PORT")
```

Comment:
Why not use `pkg/env` package for `KUBERNETES_SERVICE_PORT` as well?

Reply:
I think this env variable is defined for every container started via k8s, so it's hard for me to imagine what it would improve

Reply:
Mostly for consistency with how we handle env variables in the code base in general. Would also provide a central location if K8s ever decides to rename the env var or something like that.